book/src/repair-service.md

```diff
@@ -0,0 +1,50 @@
+# Repair Service
+
+The RepairService is in charge of retrieving missing blobs that failed to be delivered by primary communication protocols like Avalanche. It is in charge of managing the protocols described below in the `Repair Protocols` section below.  
+
+# Repair Protocols
+
+The repair protocol makes best attempts to progress the forking structure of Blocktree. 
+
+The different protocol strategies:
```

Comment:
This wording suggests you'll offer 3 competing strategies to solving the same problem, but then you describe 3 problems that the repair service addresses and one solution to each problem.  How about describing all the problems first?  That'd make it more consistent with how Anatoly writes his design proposals. That is, a design that addresses all the "challenges" in the current implementation.

Reply:
Updated as suggested